bankroll/model/activity.py

```diff
@@ -78,11 +79,30 @@ def price(self) -> Cash:
     def proceeds(self) -> Cash:
         return self.amount - self.fees
 
-    def __str__(self) -> str:
+    @property
+    def action(self) -> str:
         if self.quantity > 0:
-            action = "Buy   "
+            action = "Buy"
         elif TradeFlags.LIQUIDATED in self.flags:
             action = "Sell-L"
         else:
-            action = "Sell  "
-        return f"{self.date.date()} {action} {abs(self.quantity):>9} {self.instrument:21} {self.amount.paddedString(padding=10)} (before {self.fees.paddedString(padding=5)} in fees)"
+            action = "Sell"
+
+        return action
+
+    @staticmethod
+    def dataframeColumns() -> List[str]:
```

Comment:
I'm not a big fan of this living within the main model definition, if it sort of adds bloat without benefit to non-Pandas users. Can it be defined externally to the class in a sane way?

Reply:
I extracted this out, although the `dataframeColumnFunctions` now will have an if/elif sequence for all model types that need to be converted